cypress/support/commands.js

```diff
@@ -22,4 +22,18 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const faker = require('@faker-js/faker').faker;
+function generateUser() {
+  return {
+    firstName: faker.person.firstName(),
+    lastName: faker.person.lastName(),
+    userEmail: faker.internet.email(),
+    age: faker.number.int({ min: 1, max: 100 }),
+    salary: faker.number.int({ min: 1, max: 100000 }),
+    department: faker.company.name()
+  };
+}
+
+module.exports = { generateUser };
```

Comment:
The 'generateUser' function is correctly defined and exported. Make sure to use this function in your tests to generate random user data.